app/services/cocina_generator/structural/file_generator.rb

```diff
@@ -33,22 +38,24 @@ def request_file_attributes
           filename:,
           access:,
           administrative:,
-          hasMimeType: blob.content_type,
+          hasMimeType: cocina_file&.hasMimeType || blob.content_type,
```

Comment:
Don't you want `hasMimeType` and `size` to be in the other direction?

Reply:
Actually, also for all of these attributes.

Reply:
I suspect we're all misunderstanding because this is so confusing.

What if you are replacing an existing file? Then there will be cocina AND a blob.

Reply:
I tested by replacing a file with one with the same name but different size / md5. The new size and md5 are reflected in the updated structural cocina.

Reply:
To correct what I said above, there would be a blob on preserved_files, but the blob has a service_name of "preservation" and does not point to an actual file. So we want to use the cocina anyway on those files.